### ISSUE TITLE
Using Kenney.io glTF assets causes 'Null check operator used on a null value'

### ISSUE THREAD

**Issue by domesticmouse** [bdero/flutter_scene#27]
Repo: https://github.com/domesticmouse/hexagon_explorer

```console
Launching lib/main.dart on macOS in debug mode...
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:arm64, id:00006000-001250C00C38801E, name:My Mac }
{ platform:macOS, arch:x86_64, id:00006000-001250C00C38801E, name:My Mac }
✓ Built build/macos/Build/Products/Debug/hexagon_explorer.app
[IMPORTANT:flutter/shell/platform/embedder/embedder_surface_metal_impeller.mm(47)] Using the Impeller rendering backend (Metal).
Connecting to VM Service at ws://127.0.0.1:50840/Qr8EL3wB7Gc=/ws
Connected to the VM Service.
flutter: Flutter Scene is not ready to render. Skipping frame.
flutter: You may wait on the Future returned by Scene.initializeStaticResources() before rendering.
flutter: Unpacking Scene (nodes: 1, textures: 1)
[ERROR:flutter/runtime/dart_vm_initializer.cc(40)] Unhandled Exception: Null check operator used on a null value
#0      Node.fromFlatbuffer (package:flutter_scene/node.dart:142:55)
#1      Node.fromAsset (package:flutter_scene/node.dart:125:12)
<asynchronous suspension>
#2      MyAppState.initState.<anonymous closure> (package:hexagon_explorer/main.dart:36:61).
<asynchronous suspension>

flutter: Flutter Scene is not ready to render. Skipping frame.
flutter: You may wait on the Future returned by Scene.initializeStaticResources() before rendering.
 ```

**Comment by bdero** [comment]
These glTFs reference a colormap texture that's missing. But even if it weren't missing, the importer only really supports embedded images at the moment, so that will need to be fixed.

Also, when the import or runtime unpack runs into texture issues, it should just gracefully fallback to a placeholder texture.

Added this behavior with https://github.com/bdero/flutter_scene/commit/b7363dd8e99dd5c91360b4f096332063c29c1398

![image](https://github.com/user-attachments/assets/3fe3c9d0-e7f2-4b1b-8eea-bf913ee0d1e3)

> flutter: Texture 0 has no embedded image. A white placeholder will be used instead.

**Comment by bdero** [comment]
I added functionality to automatically attempt to load external referenced textures from the asset bundle in https://github.com/bdero/flutter_scene/commit/f0c797f9b615bd4d7dffec7c6ae6907c10ae2fec.

So these assets are now able to import properly now:
<img width="704" alt="image" src="https://github.com/user-attachments/assets/7643f856-b486-4581-9809-fe2d02877e12">

**Comment by bdero** [comment]
This is available in flutter_scene 0.5.0-0 + flutter_scene_importer 0.2.0-0.
